### PR TITLE
STUDENT_AGE_DICTIONARY() output floats for G1G2G3 & StudyTime

### PR DESCRIPTION
**Pull request by InochiFumetsu**

### CODE REVIEW

**Review by Hey1121 — APPROVED** [review]

File: T012_M1_load_data.py [hunk 1]
```diff
@@ -106,19 +106,19 @@ def student_health_dictionary(file_name: str) -> Dict[int or str, List[dict]]:
...
+            student_dict_keys[7]: float(student_stat[7]),
+            student_dict_keys[8]: float(student_stat[8]),
         })
         file_content.close()
     return health_dictionary
   
   
 
-def student_age_dictionary(file_name: str) -> Dict[int or str, List[dict]]:
+def student_age_dictionary(file_name: str) -> dict:
```
Comment (InochiFumetsu) [inline T012_M1_load_data.py:121]:
```suggestion
def student_age_dictionary(file_name: str) -> Dict[int or str, List[dict]]:
```

File: T012_M1_load_data.py [hunk 2]
```diff
@@ -223,7 +230,7 @@ def student_failures_dictionary(filename: str) -> Dict[int or str, List[dict]]:
 
     keys = []  # List for they keys of the csv file
```
Comment (InochiFumetsu) [inline T012_M1_load_data.py:231]:
```suggestion
```
unnecessary (see below suggestion)

File: T012_M1_load_data.py [hunk 3]
```diff
@@ -223,7 +230,7 @@ def student_failures_dictionary(filename: str) -> Dict[int or str, List[dict]]:
...
     # Seperate the keys with commas and remove the new line at the end
-    basic_import[0] = basic_import[0].strip('\n').split(sep=',')
+    basic_import[0] = basic_import[0].strip('\n ').split(sep=',')
     # For loop that loops through the first line of the csv file and appends the keys
     for i in basic_import[0]:
```
Comment (InochiFumetsu) [inline T012_M1_load_data.py:235]:
```suggestion
```
unnecessary (see below suggestion)

File: T012_M1_load_data.py [hunk 4]
```diff
@@ -223,7 +230,7 @@ def student_failures_dictionary(filename: str) -> Dict[int or str, List[dict]]:
...
         keys.append(i)
```
Comment (InochiFumetsu) [inline T012_M1_load_data.py:236]:
```suggestion
keys = basic_import[0][:]
```
loop unnecessary. basic_import[0] is a list of headers at this point (line 233). You do not need to iterate over that list adding one element at a time to create an identical list in a different variable name. (The second variable name isn't even necessary, but I understand wanting to do that for readability). Anyway, to create a clone of a list (referring to a separate object in memory - which is what you seem to want to do here) we were taught two different sytaxes:
    1) keys = basic_import[0].copy()
    2) keys = basic_import[0][:]

**Review by Jackinator27 — APPROVED** [review]

**Review by MilanxDjordjevic — APPROVED** [review]

**Review by Hey1121 — APPROVED** [review]

**Pull request merged**